src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/PopularComponents.razor

```diff
@@ -47,7 +47,8 @@
                     <BitFileUpload IsMultiSelect="true"
                            Label="Select files"
                            UploadUrl="@UploadUrl"
-                           RemoveUrl="@RemoveUrl">
+                           RemoveUrl="@RemoveUrl"
+                           Style="width:100%; max-width:none;">
```

Comment:
since the parent element is using the flexbox, it's better to use `flex-grow:1` instead of `width:100%`
and also, can't we use a CSS class instead of an inline style?
@maryamhdr what do you think?

Reply:
yeah, you are right about the `max-width`, but my concern is about the `width` property.

Reply:
and about the inline style, I'm ok with using it if there are some other cases on the same page.
@maryamhdr can enlighten us :)